sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -205,7 +245,9 @@ def test_spectral_embedding_callable_affinity(X, seed=36):
 @pytest.mark.filterwarnings(
     "ignore:scipy.linalg.pinv2 is deprecated:DeprecationWarning:pyamg.*"
 )
-def test_spectral_embedding_amg_solver(seed=36):
+@pytest.mark.parametrize("dtype", (np.float32, np.float64))
+def test_spectral_embedding_amg_solver(dtype):
+    seed = 36
```

Comment:
I think it's fine to keep seed as a kwarg of the test. While we cannot afford to run the CI with with many different seed values, keeping the seed as a kwargs makes it easier to locally add a line such as 

```python
@pytest.mark.parametrize("seed", range(10))
```

locally to run the test with more seeds to check that it's not too sensitive to a specific choice of the seed value. Otherwise it's a sign that the test is brittle and can fail for no good reason on different platforms (Operation System / CPU architecture / user land bitness...).

Reply:
done